### PR TITLE
Add spring security

### PR DESCRIPTION
**Pull request by Daniil9596**
Added basic support of Spring Security:
1) Getting user's details from database is achieved.
2) Login step on start is implemented.
> If you Login successfully, you will see index page, else you will redirect to registration page.
3) Logout button has been added to index page.

### CODE REVIEW

File: src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java [hunk 1]
```diff
@@ -85,4 +85,40 @@ public User findByLogin(User user) {
...
+            ResultSet rs = ps.executeQuery();
+
+            if (rs.next()) {
+                user.setId(rs.getInt(1));
+                user.setLogin(login);
+                user.setPassword(rs.getString(3));
+                user.setRoleId(rs.getInt(4));
+                user.setName(rs.getString(5));
+
+                System.out.println("===USR INFO===");
```
Comment (nikandtos1) [inline src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java]: Dont use sout for loggin

Comment (nikandtos1) [reply src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java]: And never logg any passwords))

File: src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java [hunk 2]
```diff
@@ -85,4 +85,40 @@ public User findByLogin(User user) {
         System.out.println("user is not exist");
         return userFind;
     }
+
+    @Override
+    public User getUserByLogin(String login) {
```
Comment (nikandtos1) [inline src/main/java/com/epam/lab/paymentsystem/dao/impl/UserDAOImpl.java]: We have two methods with the same logic, but different arguments. Can we reuse code and wrap one into other?

**Comment by Daniil9596** [comment]
1) Wrapped getUserByLogin() into findByLogin().
2) Remove overlooked logging of user and pwd.

**Pull request merged**